Makefile

```diff
@@ -331,6 +355,7 @@ deploy/integreatly-rhmi-cr.yml:
 	sed "s/USE_CLUSTER_STORAGE/$(USE_CLUSTER_STORAGE)/g" > deploy/integreatly-rhmi-cr.yml
 	@-oc create -f deploy/integreatly-rhmi-cr.yml
 
+#TODO: new target required for rhoam
```

Comment:
I think this todo is out of date. Am I right in saying we wont need to add a new target but we'll update delorean to have a new flag which we can set the installatin type by?

Reply:
I was nopeing to push this work off to a different ticket. Currently progressing the changes needed. So yes this todo is now out of date and is been worked on.

Reply:
Waiting on this PR https://github.com/integr8ly/delorean/pull/148 to be merged, bu the change has been put in place